### ISSUE TITLE
Ramp-up Facebook app and connect health bot to the facebook app

### ISSUE THREAD

**Issue by idigra** [dev-a-better-world/vacgaps#21]
Full instruction page for connection MS bot to FB:
https://docs.microsoft.com/en-us/azure/bot-service/bot-service-channel-connect-facebook?view=azure-bot-service-4.0#connect-a-bot-to-facebook-messenger

Minimized instructions for health bot:
https://docs.microsoft.com/en-us/healthbot/channels/facebook

**Comment by idigra** [comment]
I followed the instructions of the first link above. When trying to chat with the page, there's an error message in FB messenger.
I forgot step 3 in "Generate a Page Access Token" to grant the app pages_messaging permissions. I now tried to do it and it seems to be a nightmare that requires FB approvals, deciding privacy policies and etc.
We decided to start with a non FB bot in parallel (there's another task on that) just to unblock.
